### ISSUE TITLE
Component composition with dot notation is not supported.

### ISSUE THREAD

**Issue by sandertar** [ony3000/prettier-plugin-classnames#47]
## Dependency information
TypeScript
React
Prettier 3.2
prettier-plugin-classnames 0.6.3

## Steps to reproduce

```javascript
export const Overlay = () => {
  return (
    <Dialog.Overlay className="bg-popover backdrop-opacity-disabled focus:ring-0 fixed inset-0 z-50 h-screen w-screen" />
  );
};
```

## The current behavior
The className of the Dialog.Overlay is completely ignored.

## The expected behavior
The component classes wrapped based on the printWidth option.

**Comment by ony3000** [comment]
Hello! Thanks for reporting the issue!

I think this is a side effect caused by #46.

Until this issue is fixed, you can use `v0.6.2` instead. :pray:

**Comment by thype-it** [comment]
Thanks that fixed it for me as well

**Comment by ony3000** [comment]
@sandertar Fixed in `v0.6.4`, check it out!
